From 554a524091c78ea47dd80ff3513cab627f90d20a Mon Sep 17 00:00:00 2001
From: Kirill Radzikhovskyy <kirillrdy@gmail.com>
Date: Sun, 23 Nov 2025 06:57:53 +1100
Subject: [PATCH] awscli2: fix prompt-toolkit source override

Update the hash and add the `tag` attribute for the prompt-toolkit 3.0.51 override.
---
 pkgs/by-name/aw/awscli2/package.nix | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pkgs/by-name/aw/awscli2/package.nix b/pkgs/by-name/aw/awscli2/package.nix
index 99bac8ec2404a2c054d56c0cc0bc0fb493a649f3..3b43fb39bc9404636480e486a6953f39b5b13b6f 100644
--- a/pkgs/by-name/aw/awscli2/package.nix
+++ b/pkgs/by-name/aw/awscli2/package.nix
@@ -26,8 +26,8 @@ let
         prompt-toolkit = prev.prompt-toolkit.overridePythonAttrs (prev: rec {
           version = "3.0.51";
           src = prev.src.override {
-            inherit version;
-            hash = "sha256-kxoWLjsn/JDIbxtIux+yxSjCdhR15XycBt4TMRx7VO0=";
+            tag = version;
+            hash = "sha256-pNYmjAgnP9nK40VS/qvPR3g+809Yra2ISASWJDdQKrU=";
           };
         });
         python-dateutil = prev.python-dateutil.overridePythonAttrs (prev: rec {
